docs/aws-config.md

```diff
@@ -7,7 +7,7 @@ To set up Ark on AWS, you:
 * Configure the server
 * Create a Secret for your credentials
 
-If you do not have the `aws` CLI locally installed, follow the [user guide][5] to set it up.
+If you do not have the AWS CLI locally installed, follow the [user guide][5] to set it up.
```

Comment:
I think I prefer the previous version, or maybe "AWS CLI client".

Reply:
I'll revert for now and we can discuss later. Bikeshedding not needed :-)

Reply:
+1 to @nrb's comment